public/js/test/mochitest/head.js

```diff
@@ -198,12 +198,16 @@ function findSource(dbg, url) {
   return source.toJS();
 }
 
-function selectSource(dbg, url) {
+function selectSource(dbg, url, line) {
```

Comment:
That would be wrong though; `line` should be `undefined` when you aren't selecting a line and you don't want to jump. When we maintain scroll positions, it'll just render the source in the place where it was and shouldn't jump anywhere.